smrf/framework/model_framework.py

```diff
@@ -36,15 +36,15 @@
 from inicheck.config import UserConfig
 from inicheck.output import generate_config, print_config_report
 from inicheck.tools import check_config, get_user_config
+from smrf.envphys import sunang
+from smrf.utils import queue
```

Comment:
Why move these? If you run `make isort`, it will probably move them back.

Reply:
That was the result of `isort`.
Running it with this version:
```shell
isort -c -v smrf/framework/model_framework.py
SUCCESS: /data/projects/ARS/smrf/smrf/framework/model_framework.py Everything Looks Good!
```

Reply:
Well then, that is odd.

Reply:
Was also surprised to see that as part of the commit